extensions/amp-story-shopping/0.1/test/test-amp-story-shopping-config.js

```diff
@@ -14,9 +23,22 @@ describes.realWin(
     let win;
     let element;
     let shoppingConfig;
+    let storeService;
 
     beforeEach(async () => {
       win = env.win;
+      storeService = getStoreService(win);
+      registerServiceBuilder(win, 'story-store', function () {
+        return storeService;
+      });
+
+      const productIDtoProduct = {};
+
+      for (const item of configData['items']) {
+        productIDtoProduct[item['product-tag-id']] = item;
+      }
+
+      storeService.dispatch(Action.ADD_SHOPPING_STATE, productIDtoProduct);
```

Comment:
This is mimicking an implementation of the component, we need to remove this from the test.

Tests should provide input and then test output. 
In this way implementation can change and the test will still work.